backend/trustymain/app.go

```diff
@@ -241,6 +269,10 @@ func (a *App) loadConfig() error {
 	flags.cpu = app.Flag("cpu", "enable CPU profiling, specify a file to store CPU profiling info").String()
 	flags.isStderr = app.Flag("std", "output logs to stderr").Bool()
 	flags.dryRun = app.Flag("dry-run", "verify config etc, and do not start the service").Bool()
+	flags.hsmCfg = app.Flag("hsm-cfg", "location of the HSM configuration file").String()
+	flags.cryptoProvs = app.Flag("crypto-prov", "path to additional Crypto provider configurations").Strings()
+	flags.clientURLs = app.Flag("client-listen-url", "URL for the clients listening end-point").Strings()
+	flags.healthURLs = app.Flag("health-listen-url", "URL for the health listening end-point").Strings()
```

Comment:
Same here. The variable name is plural while the flag name is singular.

Reply:
it is plural. easier to see with `trusty --help`
the usage:
```
--client-listen-url=foo --client-listen-url=bar
```